### ISSUE TITLE
profile card/page

### ISSUE THREAD

**Issue by JTFouquier** [JTFouquier/mark2cure#56]
profile card

* username
* level
* points
* join date
* last visited date
* motivational quote
* num docs processed
* country flag
*  num annotations

Later...

* average rating

----------------------------------------
- Bitbucket: https://bitbucket.org/sulab/mark2cure/issue/56
- Originally reported by: [Andrew Su](http://bitbucket.org/asu)
- Originally created at: 2015-01-29T00:34:10.307

**Comment by JTFouquier** [comment]
Public profile // Addresses #56


→ <<cset e9ca8d16df30>>

----------------------------------------
Original comment by: [Max Nanis](http://bitbucket.org/x0xMaximus)

**Comment by JTFouquier** [comment]
Adding the cards is turning out to be far more difficult than I imagined. I overlooked the fact that they're coming from the quest view which pulls in the results dynamically in a single page. This means that b/c a refresh never occurs, lingering popovers remain, and kill events for the new one. It's doable but I want to move on to the more impactful issues now and will get back to this with a clean head.

When working well, they appear as such:
![](http://www.puff.me.uk/ss/F3yOpaytsWpwL.png)


----------------------------------------
Original comment by: [Max Nanis](http://bitbucket.org/x0xMaximus)
